docs/whats-new/staging-release-notes.md

```diff
@@ -2,10 +2,40 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 28 September 2022
```

Comment:
need to duplicate this for production release on 6 oct?

Reply:
Included for the Oct 6th prod release.